src/jit.rs

```diff
@@ -1688,9 +1661,25 @@ impl JitCompiler {
         // There is no `VerifierError::JumpToMiddleOfLDDW` for `call imm` so patch it here
         let call_unsupported_instruction = self.anchors[ANCHOR_CALL_UNSUPPORTED_INSTRUCTION] as usize;
         let callx_unsupported_instruction = self.anchors[ANCHOR_CALLX_UNSUPPORTED_INSTRUCTION] as usize;
-        for offset in self.result.pc_section.iter_mut() {
-            if *offset == call_unsupported_instruction {
-                *offset = callx_unsupported_instruction;
+        if self.config.static_syscalls {
+            let mut prev_pc = 0;
```

Comment:
wouldn't it be faster to memset the pc_section slice when we mmap, before we set the valid entries?

Reply:
During compilation the `pc_section` is used for relocation of jumps & calls. Only at the end it is steamrolled for the most part to be used at runtime.

Reply:
oh true yeah fair enough